### PR TITLE
[rubocop] Autocorrect

### PR DESCRIPTION
**Pull request by donaldong**

### CODE REVIEW

**Comment by donaldong** [comment]
update

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/4a0e1e1715552dd997e3d536394a99b6a6b9b694..8440870205f962cf14409c3bc12591c7d8425c28).

**Comment by donaldong** [comment]
fix

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/1409125f8fc96fa0a45875f6b5380c89d829a042..3653b84ac8d4f3af0423d940abf539aa65a72142).

File: lib/redis_memo/redis.rb [hunk 1]
```diff
@@ -43,9 +44,9 @@ def initialize(orig_options)
       super(primary_option)
     end
 
-    alias_method :get_primary, :get
-    alias_method :mget_primary, :mget
-    alias_method :mapped_mget_primary, :mapped_mget
+    alias get_primary get
+    alias mget_primary mget
+    alias mapped_mget_primary mapped_mget
```
Comment (katyho) [inline lib/redis_memo/redis.rb:49]: Is this a rubocop suggestion? I did a quick google search, and it feels like alias_method is a bit more straightforward to use/more predictable

File: lib/redis_memo/testing.rb [hunk 2]
```diff
@@ -5,13 +5,12 @@
 # to be more robust when testing their code that uses redis-memo.
 module RedisMemo
   class Testing
-
-    def self.__test_mode
-      @__test_mode
+    class << self
+      attr_reader :__test_mode
```
Comment (katyho) [inline lib/redis_memo/testing.rb:9]: Should we just use attr accessor here?

**Review by katyho — APPROVED** [review]

File: lib/redis_memo/memoizable/dependency.rb [hunk 3]
```diff
@@ -12,13 +12,14 @@ def memos
     @nodes.values
   end
 
-  def depends_on(dependency, **conditions)
+  def depends_on(dependency, **_conditions)
```
[the remark this answers is not in the document]
Comment (donaldong) [reply lib/redis_memo/memoizable/dependency.rb:15]: good point!

File: lib/redis_memo/redis.rb (hunk 1 above)
Comment (donaldong) [reply lib/redis_memo/redis.rb:49]: Yes it is. I agree with you tho! Let me undo this change and adjust rubocop

File: lib/redis_memo/testing.rb (hunk 2 above)
Comment (donaldong) [reply lib/redis_memo/testing.rb:9]: huh... thanks! good catch

**Comment by donaldong** [comment]
update

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/07312cdd357dca139213dc472067dd72b59da970..cd55c9971d978d93866e3ffd0cb1182793b2ed50).

**Comment by donaldong** [comment]
fix spec

:mag_right: [Click to see changes since last update](https://github.com/chanzuckerberg/redis-memo/compare/5bff904a4a27d91579d9a661c4270b7ce914b69c..c68072d2d8ffa9549f3e42b0a4518ca0377c9f5c).

**Pull request merged**